activities.js

```diff
@@ -0,0 +1,61 @@
+// List to store activities
+const activities = [];
+
+// Dictionary to store productivity scores for different categories
+const productivityScores = {
+    'Work': 10,
+    'Exercise': 8,
+    'Leisure': 5,
+    'Learning': 9,
+    'Other': 4
+};
+
+function addActivity(name, category, duration) {
+    const date = new Date().toISOString().split('T')[0];
```

Comment:
Tip: an alternative way would be to use [`toDateString()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toDateString), but that however result in a slighly different date format.